libs/core/src/components/pds-box/pds-box.tsx

```diff
@@ -120,6 +126,18 @@ export class PdsBox {
   */
   @Prop() padding?: BoxTShirtSizeType;
 
+  /**
+   * Defines the top and bottom spacing.
+   * @defaultValue none
+  */
+  @Prop() paddingBlock?: BoxTShirtSizeType;
+
+  /**
+   * Defines the left and right spacing.
+   * @defaultValue none
+  */
+  @Prop() paddingInline?: BoxTShirtSizeType;
+
```

Comment:
Can we document this to discuss how this will cascade down should the customer use all three props?

e.g
padding="100px", padding-block="50px", padding-inline="25px".  

if all three are applied, we should state that padding-block and padding-inline will override the value set for `padding`.